src/ex2/item_manager.js

```diff
@@ -0,0 +1,53 @@
+import PokemonClient from "./PokemonClient.js";
+
+class ItemManager {
+	constructor() {
+		this.id = 0;
+		this.itemArray = [];
+		this.pokemonClient = new PokemonClient();
+	}
+	addListItem(item) {
```

Comment:
Adding a space between the end of the previous function and the beginning of the next one will make the code a lot more readable (do it everywhere, not just here)